xayn-ai/src/embedding/utils.rs

```diff
@@ -53,9 +54,17 @@ where
     mean
 }
 
+/// See [`pairwise_cosine_similarity`] for details
+pub const MAXIMUM_COSINE_SIMILARITY: f32 = 1.0;
+pub const MINIMUM_COSINE_SIMILARITY: f32 = -1.0;
+pub const COSINE_SIMILARITY_RANGE: RangeInclusive<f32> =
+    MINIMUM_COSINE_SIMILARITY..=MAXIMUM_COSINE_SIMILARITY;
```

Comment:
it's a good idea to state them as constants. now we can also use them in the clamping in l.99.